Index: hei.ttx
===================================================================
--- a/hei.ttx
+++ b/hei.ttx
@@ -65644,22 +65644,22 @@
       © 2014-2019 Adobe (http://www.adobe.com/), with Reserved Font Name 'Source'.
     </namerecord>
     <namerecord nameID="1" platformID="3" platEncID="1" langID="0x409">
-      Source Han Sans SC
+      Hei
     </namerecord>
     <namerecord nameID="2" platformID="3" platEncID="1" langID="0x409">
       Regular
     </namerecord>
     <namerecord nameID="3" platformID="3" platEncID="1" langID="0x409">
-      2.001;ADBO;SourceHanSansSC-Regular;ADOBE
+      2.001;ADBO;Hei-Regular;ADOBE
     </namerecord>
     <namerecord nameID="4" platformID="3" platEncID="1" langID="0x409">
-      Source Han Sans SC
+      Hei
     </namerecord>
     <namerecord nameID="5" platformID="3" platEncID="1" langID="0x409">
       Version 2.001;hotconv 1.0.107;makeotfexe 2.5.65593
     </namerecord>
     <namerecord nameID="6" platformID="3" platEncID="1" langID="0x409">
-      SourceHanSansSC-Regular
+      Hei-Regular
     </namerecord>
     <namerecord nameID="7" platformID="3" platEncID="1" langID="0x409">
       Source is a trademark of Adobe in the United States and/or other countries.
@@ -65683,13 +65683,13 @@
       http://scripts.sil.org/OFL
     </namerecord>
     <namerecord nameID="1" platformID="3" platEncID="1" langID="0x804">
-      思源黑体
+      黑体
     </namerecord>
     <namerecord nameID="2" platformID="3" platEncID="1" langID="0x804">
       Regular
     </namerecord>
     <namerecord nameID="4" platformID="3" platEncID="1" langID="0x804">
-      思源黑体
+      黑体
     </namerecord>
   </name>
 
